src/sheet_metal_conus.py

```diff
@@ -3,188 +3,338 @@
 # Distributed under the terms of the GNU Lesser General Public License v3.0
 
 import sys
-sys.path.append('/usr/share/inkscape/extensions')
+#sys.path.append('/usr/share/inkscape/extensions')
```

Comment:
If follow instruction in install (new additions) then always get it in right folder on user's system. So will not need this work around...